src/App.js

```diff
@@ -1,22 +1,22 @@
 import React from 'react';
-import { Router, Switch, Route } from 'react-router-dom';
-import { createBrowserHistory } from 'history';
-import { MaterialPage, LoanPage, LostPage } from './pages';
+import { Router } from 'react-router-dom';
+import { useDispatch } from 'react-redux';
+import history from './services/history';
 import { Header } from './components';
 import GlobalStyle from './styles/GlobalStyle';
+import Routes from './routes';
 
-const history = createBrowserHistory();
-
-const App = () => (
-  <Router history={history}>
-    <GlobalStyle />
-    <Header />
-    <Switch>
-      <Route path="/" exact component={MaterialPage} />
-      <Route path="/loans" component={LoanPage} />
-      <Route path="/losts" component={LostPage} />
-    </Switch>
-  </Router>
-);
+const App = () => {
+  const dispatch = useDispatch();
+  dispatch({ type: 'FETCH_LOAN_REQUESTED' });
+  dispatch({ type: 'FETCH_MATERIAL_REQUESTED' });
```

Comment:
Generally we don't fetching irrelavant data such as loan and material, because management of state it is usually used on data that we use throughout the app.

Reply:
So we just fetch them into the component.